platform/src/main/java/org/hillview/sketches/SampleHeavyHittersSketch.java

```diff
@@ -1,22 +1,19 @@
 package org.hillview.sketches;
 
-import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
 import it.unimi.dsi.fastutil.ints.Int2ObjectOpenCustomHashMap;
-import it.unimi.dsi.fastutil.ints.IntHash;
 import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
-import it.unimi.dsi.fastutil.objects.Object2ObjectMap;
-import it.unimi.dsi.fastutil.objects.ObjectIterator;
 import org.hillview.dataset.api.ISketch;
 import org.hillview.table.Schema;
 import org.hillview.table.api.IMembershipSet;
 import org.hillview.table.api.IRowIterator;
 import org.hillview.table.api.ITable;
 import org.hillview.table.rows.RowSnapshot;
-import org.hillview.table.rows.VirtualRowSnapshot;
+import org.hillview.table.rows.VirtualRowHashStrategy;
 import org.hillview.utils.MutableInteger;
 
 import javax.annotation.Nullable;
-import java.util.List;
+
+import static org.hillview.sketches.FreqKList.getUnion;
```

Comment:
I don't like static imports, I would prefer for you just to call the class explicitly below: FreqKList.getUnion(). It is also more readable, since it makes it clear in the code where the function is coming from.